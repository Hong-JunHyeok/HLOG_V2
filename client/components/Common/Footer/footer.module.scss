@use "../../../styles/colors.module.scss" as colors;

.container {
  display: flex;
  justify-content: center;
  padding: 2rem 0;
  background-color: colors.$normal_text;
  color: colors.$white_text;

  .logo {
    line-height: 3rem;
    font-size: 26px;
    font-weight: lighter;
    font-family: ChosunBg;
  }
}
