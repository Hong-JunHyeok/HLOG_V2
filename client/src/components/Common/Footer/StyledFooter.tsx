import styled from '@emotion/styled';
import ColorSet from '@/styles/colorSet';
import mediaQueryHelper from '@/styles/mediaHelper'

const CssVariable = {}

const StyledFooter = {
  FooterContainer: styled.footer`
    display: flex;
    align-items: center;
    justify-content: center;
    padding: 2rem 0;
    background-color: ${ColorSet['--grey800']};
    color: ${ColorSet['--white']};
  `,
  FooterLogo: styled.span`
    line-height: 3rem;
    font-size: 26px;
    font-weight: lighter;
    padding: 0 .5rem;
  `
}

export default StyledFooter;
