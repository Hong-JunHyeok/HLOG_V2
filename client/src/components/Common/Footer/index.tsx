import React from "react";
import S from './StyledFooter';

interface FooterProps {}

const Footer: React.FunctionComponent<FooterProps> = () => {
  return (
    <S.FooterContainer>
      <span>All Copyright reserved</span>
      <S.FooterLogo>HLOG</S.FooterLogo>
    </S.FooterContainer>
  );
};

export default Footer;
