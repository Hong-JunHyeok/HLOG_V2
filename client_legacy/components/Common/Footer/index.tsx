import React from "react";
import styles from "./footer.module.scss";

interface FooterProps {}

const Footer: React.FunctionComponent<FooterProps> = () => {
  return (
    <React.Fragment>
      <footer className={styles.container}>
        <p>
          All Copyright reserved <span className={styles.logo}>HLOG</span>
        </p>
      </footer>
    </React.Fragment>
  );
};

export default Footer;
