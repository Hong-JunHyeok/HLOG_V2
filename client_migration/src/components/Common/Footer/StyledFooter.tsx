import styled from '@emotion/styled';
import ColorSet from '@/styles/colorSet';
import mediaQueryHelper from '@/styles/mediaHelper'

const CssVariable = {}

const StyledFooter = {
  FooterContainer: styled.footer`
    display: flex;
    justify-content: center;
    padding: 2rem 0;
    background-color: ${ColorSet['--grey800']};
    color: ${ColorSet['--white']};
  `,
  FooterLogo: styled.span`
    line-height: 3rem;
    font-size: 26px;
    font-weight: lighter;
  `
}

export default StyledFooter;
