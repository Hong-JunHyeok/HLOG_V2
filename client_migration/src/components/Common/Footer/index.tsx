// import React from "react";

// interface FooterProps {}

// const Footer: React.FunctionComponent<FooterProps> = () => {
//   return (
//     <React.Fragment>
//       <footer>
//         <p>
//           All Copyright reserved <span className={styles.logo}>HLOG</span>
//         </p>
//       </footer>
//     </React.Fragment>
//   );
// };

// export default Footer;
