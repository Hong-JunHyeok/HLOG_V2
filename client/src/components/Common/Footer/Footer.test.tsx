import React from 'react';
import { render } from '@testing-library/react';

import Footer from '.';

describe('렌더링 테스트', () => {
  it('Footer 컴포넌트가 기존의 스냅샷과 동일하다', () => {
    const utils = render(<Footer />);
    expect(utils.container).toMatchSnapshot();
  });
  it('Footer 컴포넌트에 텍스트가 잘 렌더링 된다.', () => {
    const { getByText } = render(<Footer />);
    expect(getByText('HLOG')).toBeDefined();
  });
});
