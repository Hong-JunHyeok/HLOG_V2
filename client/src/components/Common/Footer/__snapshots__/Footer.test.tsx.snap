// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`렌더링 테스트 Footer 컴포넌트가 기존의 스냅샷과 동일하다 1`] = `
<div>
  <footer
    class="css-z7xpex"
  >
    <span>
      All Copyright reserved
    </span>
    <span
      class="css-1g39gi5"
    >
      HLOG
    </span>
  </footer>
</div>
`;
